### PR TITLE
Update Grid rest api to use actix web 4 and Splinter 0.6

### PR DESCRIPTION
**Pull request by dnewh**
To test creating a splinter circuit, follow this tutorial with the updates indicated below: https://grid.hyperledger.org/docs/0.3/creating_splinter_circuits.html. 

Step 4: Use the following command to propose a circuit: 
```
splinter circuit propose \
--node alpha-node-000::tcps://splinterd-alpha:8044 \
--node beta-node-000::tcps://splinterd-beta:8044 \
--service gsAA::alpha-node-000 --service gsBB::beta-node-000 \
--service-type gsAA::scabbard --service-type gsBB::scabbard \
--service-arg gsAA::admin_keys=$(cat gridd.pub) \
--service-arg gsBB::admin_keys=$(cat gridd.pub) \
--service-peer-group gsAA,gsBB --auth-type trust \
--management grid
```

from here, you should be able to continue as normal.

### CODE REVIEW

File: daemon/src/rest_api/mod.rs [hunk 1]
```diff
@@ -738,139 +805,144 @@ mod test {
...
         assert_eq!(agent.org_id, KEY2.to_string());
         assert_eq!(agent.service_id, Some(TEST_SERVICE_ID.to_string()));
     }
 
-    ///
-    /// Verifies a GET /organization responds with an Ok response
-    ///     with an empty organization table
-    ///
-    #[actix_rt::test]
+    // ///
```
[the remark this answers is not in the document]
Comment (dnewh) [reply daemon/src/rest_api/mod.rs]: updated

**Comment by leebradley** [comment]
Getting the following on `gridd-alpha` in the Splinter demo

```
gridd-alpha                        | DEBUG [reqwest::connect] starting new connection: http://splinterd-alpha:8085/
gridd-alpha                        | thread 'actix-rt|system:0|arbiter:1' panicked at 'not currently running on a Tokio 0.2.x runtime.', /root/.cargo/registry/src/github.com-1ecc6299db9ec823/tokio-0.2.25/src/runtime/handle.rs:118:28
```

When I try to create an organization using the following command:

```
gridd-alpha grid organization create \
  $ORG_ID $ORG_FULLNAME \
  --alternate-ids $ORG_ALTID
```

**Review by leebradley — APPROVED** [review]
LGTM 👍

Tested that I'm able to setup a circuit and hit endpoints.

We'll want to update the Grid docs to reflect the `--auth-type trust` necessary now when creating the splinter circuit.

**Review by CLEckhardt — APPROVED** [review]
tested and lgtm

**Review by vaporos — APPROVED** [review]

**Pull request merged**